client/layout/index.jsx

```diff
@@ -159,6 +159,7 @@ const Layout = createReactClass( {
 			<div className={ sectionClass }>
 				<DocumentHead />
 				<SitesListNotices />
+				<QuerySites primaryAndRecent />
```

Comment:
Funny idea: will it work if you combine the flags? `<QuerySites allSites primaryAndRecent />`.

Reply:
Yes, it should. But I wanted to have two of them so it's easier to reason about.